packages/web-components/src/toolbar/toolbar.spec.md

```diff
@@ -0,0 +1,88 @@
+# Toolbar
+
+## Component Description
+
+A toolbar is a container for grouping a set of controls, such as buttons, menubuttons, or checkboxes.
+
+When a set of controls is visually presented as a group, the toolbar role can be used to communicate the presence and purpose of the grouping to screen reader users. Grouping controls into toolbars can also be an effective way of reducing the number of tab stops in the keyboard interface.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/QfHyx6ApwFEcw5q4pM9CSZ/Toolbar?node-id=2351-6672&t=MuajdFxoCBjwlK4w-0)
+
+[Link to Fluent 2 Storybook](https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-toolbar--default)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr orientation: "horizontal" "vertical" | "horizontal"
+- @attr size: "small" "medium" "large" | "medium"
+
+### Outputs
+
+### Slots
+
+- label: Content which appears as the label for the toolbar.
+- start: Content which appears at the start of the toolbar, before the default slotted items.
+- default: Content which appears in the default location of the toolbar, left-aligned.
+- end: Content which appears at the end of the toolbar, right aligned.
+
+### CSS Variables
+
+None
+
+## Interactions
+
+Interactions are inherited from FAST Toolbar component.
+
+| Name             | Description                             |
+| ---------------- | --------------------------------------- |
+| `clickHandler`   | Sets the active index                   |
+| `focusinHandler` | Sets focus on element with active index |
+| `keydownHandler` | Handles keydown events on the toolbar   |
+
+## Accessibility
+
+- [x] Find the matching component through WCAG's patterns: https://www.w3.org/WAI/ARIA/apg/patterns/
+  - https://www.w3.org/WAI/ARIA/apg/patterns/toolbar.html
+- [x] Are there any accessibility elements unique to this component?
+- [x] List ARIA attributes
+  - [x] role="toolbar" in FAST
+  - [x] aria-orientation="vertical" in FAST
+- [ ] Does the component support 400% zoom?
+- [ ] What keyboard behaviors does the component support?
+  - [x] Tab : Moves focus to the next focusable element in the toolbar.
+  - [x] Shift + Tab : Moves focus to the previous focusable element in the toolbar.
+  - [x] Arrow keys : Moves focus to the next/previous focusable element in the toolbar.
+
+## Preparation
+
+- [x] Find the base FAST Component
+  - This component will inherit from [FAST Toolbar](https://explore.fast.design/components/fast-toolbar)
+- [x] Check the [Fluent UI React V9 Component Spec](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-toolbar/docs/Spec.md) for differences and document:
+
+  - Differences between FAST Toolbar component and Fluent Toolbar component spec:
```

Comment:
There is also the difference of React v9 `vertical: boolean;` and FAST Toolbar `orientation: 'vertical' | 'horizontal'` where `horizontal` is the default value. This may be an earned difference in React conventions vs Web Components conventions. I'm not sure if there is a resolution needed here or not, but we should document the difference.

Reply:
Addressed in new PR.